src/controllers/task.list.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from "express";
+import { Templates } from "../types/template.paths";
+import { initTaskList, TaskList } from "../types/task.list";
+import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
+import { getCompanyProfile } from "../services/company.profile.service";
+import { urlParams } from "../types/page.urls";
+
+export const get = async (req: Request, res: Response) => {
+  const companyNumber = req.params[urlParams.PARAM_COMPANY_NUMBER];
+  const company: CompanyProfile = await getCompanyProfile(companyNumber);
+  const taskList: TaskList = initTaskList();
+  const tradingStatus = "1";
```

Comment:
it's not clear what tradingStatus 1 means, can you define a constant or enum for it?

Reply:
I wasn't sure either I am aware of code 0,1,3 on NCS-233 but 1 and 2 have been used here and I'm not sure which relates to which so that's why I put a placeholder I for now as the task is just to have  a page in place we can navigate to. The scope of this ticket does not feel all that clear.